index.js

```diff
@@ -23,7 +24,9 @@ const main = async () => {
   const octokit = new GitHub(token)
 
   const res = await octokit.pulls.list(query)
-  const pr = res.data.length && res.data[0]
+  const pr = author ?
+    res.data.length && res.data[0].user.login === author && res.data[0] :
+    res.data.length && res.data[0]
```

Comment:
This change might not be ideal if there is more than one PR returned but the first one is not from the specified author.  I think a little modification to filter the list first would yield a slightly better result, e.g. something like:

```suggestion
  const pr = author ?
    res.data.length && res.data.filter(pr => pr.user.login === author)[0] :
    res.data.length && res.data[0]
```

Reply:
Good catch and great suggestion. 👍